test/fixtures/roles.sql

```diff
@@ -20,4 +20,17 @@ ALTER ROLE postgrest_test_authenticator SET pgrst.instance."db-prepared-statemen
 ALTER ROLE postgrest_test_authenticator SET pgrst.instance."db-pre-request" = 'prereq_func';
 ALTER ROLE postgrest_test_authenticator SET pgrst.instance."db-max-rows" = '1000';
 ALTER ROLE postgrest_test_authenticator SET pgrst.instance."db-extra-search-path" = 'public, extensions';
-ALTER ROLE postgrest_test_authenticator SET pgrst.instance."db-anon-role" = 'postgrest_test_author';
+
+-- non-reloadable configs for io tests
+ALTER ROLE postgrest_test_authenticator SET pgrst.instance."server-host" = 'modified';
```

Comment:
I suggest to change the `modified` string to `ignored` in all those cases, to make it clear that this value should never be taken.